### PR TITLE
Improvements to send_email()

### PR DESCRIPTION
**Pull request by amberylx**
* Pass `email_uuid` as event `task_id`
* Do type checking on expected list parameters `to`, `cc`, `bcc`, `reply_to`

### CODE REVIEW

**Comment by amberylx** [comment]
[Change email bucket to be an environment variable](https://trello.com/c/pMEJt4we/1582-change-email-bucket-to-be-an-environment-variable)

**Comment by amberylx** [comment]
[Improve email event logging](https://trello.com/c/wruV6eLF/1646-improve-email-event-logging)

**Comment by amberylx** [comment]
[Improve error handling of zc_common.send_email()](https://trello.com/c/RBUVjSca/1559-improve-error-handling-of-zc-common-send-email)

File: zc_common/email.py [hunk 1]
```diff
@@ -72,6 +72,19 @@ def send_email(from_email=None, to=None, cc=None, bcc=None, reply_to=None,
         with attachments {} and files {}'''
         logger.info(msg.format(email_uuid, to, from_email, attachments, files))
 
+    to = to.split(',') if isinstance(to, str) else to
```
Comment (tpilara) [inline zc_common/email.py]: Suggest changing to `isinstance(to, six.string_types)`

**Comment by amberylx** [comment]
@tpilara Incorporated the feedback, thanks.

**Pull request merged**